x-pack/plugins/alerting/public/pages/maintenance_windows/helpers/convert_from_maintenance_window.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment';
+import { has } from 'lodash';
+import { MaintenanceWindow } from '../types';
+import { EndsOptions, Frequency } from '../constants';
+import { FormProps, RecurringScheduleFormProps } from '../components/schema';
+import { getInitialByWeekday } from './get_initial_by_weekday';
+import { RRuleParams } from '../../../../common';
+
+export const convertFromMaintenanceWindow = (maintenanceWindow: MaintenanceWindow): FormProps => {
```

Comment:
nitpick: let's give this a more descriptive name, what is it converting to?

Reply:
Resolved in this commit 668904dcc0023fa6f7698689bda52a0e2541cc70